monix-catnap/shared/src/main/scala/monix/catnap/Semaphore.scala

```diff
@@ -134,10 +117,10 @@ final class Semaphore[F[_]] private (provisioned: Long, ps: PaddingStrategy)(
     *
     * @see [[tryAcquireN]] for the version that can acquire multiple permits
     * @see [[acquire]] for the version that can wait for acquisition
-    * @see [[withPermit]] the preferred way to acquire and release
+    * @see [[permit]] the preferred way to acquire and release
     */
   @UnsafeProtocol
-  override def tryAcquire: F[Boolean] = tryAcquireN(1)
+  def tryAcquire: F[Boolean]
```

Comment:
I kept signatures like that to override docs. I think monix version is better, for the most part